spec/operators/every-spec.js

```diff
@@ -0,0 +1,67 @@
+/* globals describe, it, expect, expectObservable, hot */
+var Rx = require('../../dist/cjs/Rx');
+
+describe('Observable.prototype.every()', function() {
+  function truePredicate(x) {
+    return true;
+  }
+  
+  function predicate(x) {
+      return x % 5 === 0;
+  }
+  
+  it('should return true if source is empty', function() {
```

Comment:
This test case comes from [RxJS3](https://github.com/Reactive-Extensions/RxJS/blob/master/tests/observable/every.js#L12). 

Little bit unsure if this behavior is expected, if source observable is empty then `every` will emit `true` always regardless of predicate.